backend/app/services/chat/ws_emitter.py

```
@@ -929,29 +935,51 @@ async def safe_emit_in_main_loop(
-            )
+        # No running loop, try to schedule in main loop
+        if _main_event_loop is not None and _main_event_loop.is_running():
+            try:
+                # Schedule in main loop without waiting (fire-and-forget)
+                asyncio.run_coroutine_threadsafe(
+                    emit_func(*args, **kwargs), _main_event_loop
+                )
+                logger.debug(
+                    "[safe_emit] Scheduled emit in main loop (no current loop)"
+                )
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)